server/models/database.js

```diff
@@ -0,0 +1,16 @@
+import pg from 'pg';
+
+const connection = {
+  user: 'postgres',
+  host: 'localhost',
+  database: 'mtracker',
+  password: 'sam',
```

Comment:
It's better to use `Environment Variables` for your database name and password, even though they are local credentials.